packages/minifier-js/minifier.js

```diff
@@ -29,14 +30,11 @@ meteorJsMinify = function (source) {
     // Although Babel.minify can handle a wider variety of ECMAScript
     // 2015+ syntax, it is substantially slower than UglifyJS, so we use
     // it only as a fallback.
-    if (Babel.getMinifierOptions) {
-      var options = Babel.getMinifierOptions({
-        inlineNodeEnv: NODE_ENV
-      });
-      result.code = Babel.minify(source, options).code;
-    } else {
-      result.code = Babel.minify(source).code;
-    }
+    var options = Babel.getMinifierOptions({
+      inlineNodeEnv: NODE_ENV
+    });
+    result.code = Babel.minify(source, options).code;
+    result.minifier = 'babel-minify';
```

Comment:
You're setting `result.minifier`, but where are you actually reporting the minifier used?

Reply:
The result object is returned to the standard-minifier-js (or any other plugin using minifier-js). So I use it in a fork of standard-minifier-js (printing it in a not really user friendly way). Since standard-minifier-js generates statistics, it might be useful to also generate statistics about the used minifier. However, I do not know in which way it should be reported to be useful for most users. If you have any suggestions, I might give it a try to add such statistics to standard-minifier-js.